serve/mlc_serve/engine/staging_engine_worker.py

```diff
@@ -66,7 +66,7 @@ class SequenceGenerationOutput:
     new_tokens: List[int]
     finish_reason: Optional[FinishReason] = None
     error: Optional[Union[str, ValidationError]] = None
-    logprob_info: Optional[RawLogprobsInfos] = None
+    logprob_info: Optional[RawLogprobsInfo] = None
```

Comment:
We can return list of RawLogprobsInfo, due to new_tokens is a list. Particularly it is needed for loglikelihood and speculative decoding extension in the future

Reply:
Changed to `Optional[List[RawLogprobsInfo]]`